src/commands/memory-stats.json

```diff
@@ -112,7 +112,7 @@
                 }
             },
             "patternProperties": {
-                "^db\\.\\d+$": {
+                "^db\\\\.\\\\d+$": {
```

Comment:
Im a bit lost here. Are you sure that right? What are we escaping here? Both the Json string and the c string? Maybe instead, the python code that generates the c code should add it the escaping the c code?
P.s. home come I don't see changes to the Def file?

Reply:
`-DLOG_REQ_RES` We need to specify it to change the def file of reply.

i guess there is a double escaping, seem it is python issue, i try a new way in the new commit
```python3
>>> json.dumps("^db\.\d+$")
'"^db\\\\.\\\\d+$"'
```